src/index.ts

```diff
@@ -358,12 +356,13 @@ const exStatic = serv('./', {
 	extensions: EXTENSIONS,
 	index: INDEX,
 	maxAge: MAX_AGE,
-	logger: (req, res) => reqLog(req, res as Resp),
+	logger: (req, res) => reqLog(req, res as Response),
 })
 
-const genHandler: ReqListener = (req, res) => { //? General handler
+const genHandler: RequestListener = (req, res) => { //? General handler
 	const reqPath = getPath(req.url!)
 	let path = decodeURI(reqPath)
+	if (req.method !== 'GET') notFoundHandler(req, res)
```

Comment:
Have you tested this code? I bet not because i can tell that it throws errors even running it.